__init__.py

```diff
@@ -1980,6 +1980,13 @@ class BlenderKitAddonPreferences(AddonPreferences):
         update=utils.save_prefs,
     )
 
+    sidebar_panels: BoolProperty(
+        name="Hide sidebar panels",
+        description="Hide BlenderKit sidebar panels (search, upload, and selected model functionality)",
```

Comment:
I would add warning, that this prevents asset uploads, import settings etc and that to upload any asset, one needs to disable this option again.

Maybe we could mention it as a N-panel?

Reply:
Well, as soon as we mention it, the point of the option disappears. Since the issue is one more tab is added (most addons add their own tab, that's the source of the issue..)